splitio/python-client
# Feature/mget treatments

# Python SDK

## What did you accomplish?
- Added `fetch_many` method in storages (uwsgi, redis, inmemory) to grab a dict of Split.model.
- Updated evaluator and added `evaluate_feature` and `evaluate_features` for supporting one feature and multiple feature in evaluation.
- Updated client to do multiple evaluation at once by grabbing a list of splits.
- Updated tests.

## How do we test the changes introduced in this PR?
- Check calls in redis.
- Execute `python setup.py test`

## Extra Notes
